servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -569,53 +589,131 @@ void serviceTalkToServiceTalkErrorWithStatusViaServerFilter(
 
     @ParameterizedTest
     @MethodSource("sslStreamingAndCompressionParams")
-    @Disabled("https://github.com/apple/servicetalk/issues/1489")
-    void grpcJavaToGrpcJavaTimeout(final boolean ssl,
-                                   final boolean streaming,
-                                   final String compression) throws Exception {
-        Duration timeout = Duration.ofNanos(1);
+    void grpcJavaToGrpcJavaTimeout(final boolean ssl, final boolean streaming, final String compression)
```

Comment:
For all 4 variants: consider clarifying in the test name that timeout/deadline is applied on the client-side